server/db/connection.js

```diff
@@ -24,7 +25,7 @@ class Db {
    */
   connectDb() {
     let dbOptions = {
-      dbName: "" + dbName,
+      dbName: "" + (this.isTest ? `${dbName}-test` : dbName),
```

Comment:
good thing to have a separation of DBs. Having a test db is good since we can fully control the used data without affecting the users' data itself